src/main/java/org/jabref/gui/preview/PreviewViewer.java

```diff
@@ -165,13 +159,13 @@ public PreviewViewer(BibDatabaseContext database,
         previewView.setContextMenuEnabled(false);
 
         previewView.getEngine().getLoadWorker().stateProperty().addListener((observable, oldValue, newValue) -> {
-
             if (newValue != Worker.State.SUCCEEDED) {
                 return;
             }
+
             if (!registered) {
-                stateManager.activeSearchQuery(SearchType.NORMAL_SEARCH).addListener(listener);
```

Comment:
Can we also remove the `activeSearchQuery` completely from the state manager?

Reply:
It is still used by `GlobalSearchBar` because there is only one bar shared between the open libraries.